### PR TITLE
Post registration

### PR DESCRIPTION
**Pull request by samrespass**
Here's what happened! 

- [x] Logout bug exorcised
- [x] Registration now instantly logs in new users and navigates them to open games page

that's all folks!

### CODE REVIEW

File: src/components/gamesList.js [hunk 1]
```diff
@@ -203,6 +203,77 @@ export default function GamesList({ token, games, listType, listTitle }) {
                         Show more
                     </button-showmore>
 
+<<<<<<< HEAD
```
Comment (racheldk) [inline src/components/gamesList.js]: This looks like the leftovers from a merge - could you delete the <<<<<<< HEAD and ========= please?

File: src/components/gamesList.js [hunk 2]
```diff
@@ -276,6 +347,7 @@ export default function GamesList({ token, games, listType, listTitle }) {
                                 return null;
                         }
                     })()}
+>>>>>>> 487dcc7df5a6076fbf77a808f7111edd8af81c6c
```
Comment (racheldk) [inline src/components/gamesList.js]: What does line 350 do?

Comment (racheldk) [reply src/components/gamesList.js]: I think this might also be leftover from a merge.

File: src/pages/myGamesPage.js [hunk 3]
```diff
@@ -25,9 +25,14 @@ import Footer from "../components/FooterMenu";
 import { useEffect, useState } from "react";
 import axios from "axios";
 import GamesList from "../components/gamesList";
+<<<<<<< HEAD
```
Comment (racheldk) [inline src/pages/myGamesPage.js]: Can you resolve this merge conflict please?

File: src/pages/myGamesPage.js [hunk 4]
```diff
@@ -157,6 +162,72 @@ export default function MyGames({
 
     return (
         <>
+<<<<<<< HEAD
```
Comment (racheldk) [inline src/pages/myGamesPage.js]: Another merge conflict to be resolved, please.

File: src/pages/userProfile.js [hunk 5]
```diff
@@ -70,6 +69,8 @@ const handleCloseModal = (game) => {
                   />
                 </Heading>
               </div>
+              {user[0].profile > 0 &&
```
Comment (racheldk) [inline src/pages/userProfile.js:72]: What is line 72 doing? Is this like checking for the length of an array? Checking that the user has a profile?

**Comment by racheldk** [comment]
Sam, it looks like we're holding the token in LocalStorageState in HeaderMenu, login, and registration. Does it mess anything up to have more than one token storage place? If so, can we move holding the token to App.js and pass in token and setToken to these components instead?

File: src/components/HeaderMenu.js [hunk 6]
```diff
@@ -43,7 +44,9 @@ function Header(token, setToken) {
             });
 
         }
-
+        if (!token) {
```
Comment (Carlos-Tirado) [inline src/components/HeaderMenu.js:47]: oh!  got it

File: src/pages/userProfile.js (hunk 5 above)
Comment (samrespass) [reply src/pages/userProfile.js:72]: Yep this is to avoid react tryna render anything before it's ready

**Pull request merged**